scripts/commit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# bash script to commit in accordance with the branch name
+# usage: ./scripts/commit.sh
+branch=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
+branch="${branch##*feat/}"
+branch="${branch//\_/ }"
+issue_number="${branch//Issue\-/}"
+title=$( cut -d ' ' -f 2- <<< "$issue_number" )
+issue_number=$( cut -d ' ' -f 1 <<< "$issue_number" )
+commit_message="Closes #$issue_number $title"
+git commit -m "$commit_message"
```

Comment:
The `a` flag is very useful in order to automatically stages all modified files.
Also, open-source usually means signing commits with the `s` option.

More on that [here](https://www.freecodecamp.org/news/git-commit-command-explained/#:~:text=The%20%2Da%20Option&text=The%20%2Da%20stands%20for%20all,aware%20of%20will%20be%20committed.) and [here](https://stackoverflow.com/questions/64840485/what-is-the-difference-between-git-commit-s-and-s).

```suggestion
git commit -sam "$commit_message"
```